### PR TITLE
fix(docs): show where `getNodesByType` comes from

### PR DESCRIPTION
**Pull request by kurttomlinson**
## Description

The example code showing how to proactively update nodes used `getNodesByType` but didn't destructure it from the function parameters. This PR fixes that omission.

### CODE REVIEW

**Comment by kurttomlinson** [comment]
Maybe do both? I think there's a good argument to be made for listing the
correct function parameters if we think they're important enough to be
listed out in the first place.

The reason why I made this PR is because I was following this code snippet
in my own code. I needed `getNodesByType`, and this snippet was missing it,
so I was very confused. I thought maybe it was supposed to be imported.
After digging through the docs for a while, I finally found it's a function
parameter that I needed to destructure. I was hoping to prevent other
people from running into the same situation.

**Comment by AishaBlake** [comment]
@kurttomlinson that sounds like a good solution to me! I just realized I wrote the same thing in my notes but not in my actual comment. 😅

File: docs/docs/creating-a-source-plugin.md [hunk 1]
```diff
@@ -432,7 +432,7 @@ If possible, the proactive listener approach creates the best experience if exis
 The code to support this behavior looks like this:
```
Comment (marcysutton) [inline docs/docs/creating-a-source-plugin.md:432]:
```suggestion
Here's some pseudo code that shows this behavior:
```

Comment (kurttomlinson) [reply docs/docs/creating-a-source-plugin.md]: I've committed this change. Thanks for the suggestion!

**Pull request merged**